### PR TITLE
cost accounting project created

### PR DESCRIPTION
**Pull request by Alexey-Lebedev-Exceed**

### CODE REVIEW

**Comment by EgorAryamovExceedTeam** [comment]
Good job you are the best

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+module.exports.createNewTask = ((req, res) => {
+  const task = new Task (req.body);
+  task.save().then(result => {
+    res.send(result)
+  });
+});
+
+module.exports.changeTaskInfo = ((req, res) => {
+  Task.updateOne({_id: req.body._id}, req.body).then(result => {
```
Comment (KirillGolenko) [inline costAccounting/NodeJS/src/modules/controllers/task.controller.js]: and if I send a request without ID

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+const Task = require('../../db/models/task/index');
+
+module.exports.getAllTasks = ((req, res) => {
+  Task.find().then(result => {
+    res.send(result);
+  });
+});
+
+module.exports.createNewTask = ((req, res) => {
```
Comment (KirillGolenko) [inline costAccounting/NodeJS/src/modules/controllers/task.controller.js:14]: and if I send an empty request

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+module.exports.changeTaskInfo = ((req, res) => {
+  Task.updateOne({_id: req.body._id}, req.body).then(result => {
+    Task.find({_id: req.body._id}).then(result => {
+      res.send(result)
+    });
+  });
+});
+
+module.exports.deleteTask = ((req, res) => {
+  Task.deleteOne({_id: req.query._id}).then(result => {
```
Comment (KirillGolenko) [inline costAccounting/NodeJS/src/modules/controllers/task.controller.js]: and if I send a request without ID

File: costAccounting/NodeJS/src/db/models/task/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const taskScheme = new Schema({
+  shop: String,
+  date: String,
```
Comment (usatovaDasha) [inline costAccounting/NodeJS/src/db/models/task/index.js:7]: string?

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js (hunk 2 above)
Comment (usatovaDasha) [reply costAccounting/NodeJS/src/modules/controllers/task.controller.js:14]: ????

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js [hunk 5]
```diff
@@ -2,9 +2,13 @@
 const Task = require('../../db/models/task/index');
 
 module.exports.getAllTasks = ((req, res) => {
-  Task.find().then(result => {
-    res.send(result);
-  });
+  if(!req.body){
+    console.log('Данные не верны')
```
Comment (usatovaDasha) [inline costAccounting/NodeJS/src/modules/controllers/task.controller.js]: send

File: costAccounting/NodeJS/src/modules/controllers/task.controller.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ const Task = require('../../db/models/task/index');
 
 module.exports.getAllTasks = ((req, res) => {
   if(!req.body){
-    console.log('Данные не верны')
+    res.send('Данные не верны')
```
Comment (usatovaDasha) [inline costAccounting/NodeJS/src/modules/controllers/task.controller.js]: add status for error 500, 404 ....

**Pull request merged**